docs/source/man.rst

```diff
@@ -137,14 +137,21 @@ To create a credential, supply SSH credentials as either a username-password pai
 
   Prompts for the passphrase to be used when connecting using an ssh keyfile that requires a passphrase. Can only be used with the ``--sshkeyfile`` option.
 
-``--sudo-password``
+``--become_password``
 
-  Prompts for the password to be used when running a command that uses sudo on the systems to be scanned.
+  Prompts for the privilege escalation password to be used when running an ansible playbook.
```

Comment:
Maybe something like:
`Prompts for the privilege escalation password to be used when running a network scan.`